app/assets/javascripts/solidus_afterpay/afterpay_checkout_button.js

```diff
@@ -38,6 +38,24 @@ $(document).bind("afterpay.loaded", function () {
           });
       },
       onComplete: function (data) {
+        if (data.data.status == "SUCCESS") {
+          Spree.ajax({
+            method: "POST",
+            url: `/solidus_afterpay/express_callbacks/${orderNumber}.json`,
+            data: {
+              token: data.data.orderToken,
+              payment_method_id: paymentMethodId,
+            },
+          })
+            .success(function (response) {
+              window.location.href = response.redirect_url;
+            })
+            .error(function (error) {
+              console.error(error);
```

Comment:
I think we need to handle this `error` branch instead of just logging the error in the console